src/utils.js

```diff
@@ -192,24 +192,28 @@ export function getStrictSsl() {
 }
 
 /**
- * Returns an OS platform
- * - darwin
+ * Returns an OS platform compatible with scDownload endpoint
+ * - macos
  * - linux
- * - win32
- * etc
+ * - windows
  */
 export function getPlatform() {
+  if (process.platform == 'win32') {
```

Comment:
maybe use `process.platform.startsWith("win")` condition instead, don't know why `process.platform` returns win32

Reply:
Added `isWindows()` function to check for `process.platform.startsWith("win")`